src/App.jsx

```diff
@@ -14,70 +15,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleAddButtonClick = (good) => {
+    setSelectedGood(prevGood => (prevGood === good ? null : good));
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const handleClearButtonClick = () => {
+    setSelectedGood(null);
```

Comment:
```suggestion
    setSelectedGood('');
```

Or empty line as selected good is a string